api/interestr/website/templates/login.html

```diff
@@ -0,0 +1,86 @@
+<!doctype html>
```

Comment:
I suggest not adding files you don't use in the feature you implement. So that other people who implement other features, don't get any conflicts from these files.

Reply:
You may remove these files with `git rm <filename>`. `git rm login.html` in this case